lib/process-manager.ts

```diff
@@ -124,6 +124,11 @@ export class QueryProcessWrapper<T, U> implements ProcessWrapper {
 				throw error;
 			}
 
+			if (message.startsWith(`SLOW\n`)) {
+				Monitor.slow(message.slice(5));
```

Comment:
`Monitor` is not a Node.js global and can't be called from inside PM.

What we might want is to set up a listener inside PM for custom debug messages, which we could customize from the server.